NDTensors/src/lib/GradedAxes/src/unitrangedual.jl

```diff
@@ -6,7 +6,10 @@ UnitRangeDual(a::AbstractUnitRange) = UnitRangeDual{eltype(a),typeof(a)}(a)
 dual(a::AbstractUnitRange) = UnitRangeDual(a)
 nondual(a::UnitRangeDual) = a.nondual_unitrange
 dual(a::UnitRangeDual) = nondual(a)
+flip(a::UnitRangeDual) = dual(flip(nondual(a)))
 nondual(a::AbstractUnitRange) = a
+isdual(::AbstractGradedUnitRange) = false
```

Comment:
```suggestion
isdual(::AbstractUnitRange) = false
```
seems like a better definition since as we've discussed we are allowing any unit ranges to have the concept of dual or non-dual.

Reply:
Done. This will be treated more carefully in #1531